Update fix-issues-with-printers-scanners-and-lob-applications-that-se… (#2648)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md

3. Check that your device or application supports TLS version 1.2 or above. The best way to check is by upgrading the firmware on the device or updating the application you're sending email from to the latest version. Contact your device manufacturer to confirm that it supports TLS version 1.2 or above.

### Error: Authentication unsuccessful

> If you are getting one of the following erros- 

Reviewer: ```suggestion
If you are getting one of the following errors: 
```

> 535 5.7.3 Authentication unsuccessful

Reviewer: ```suggestion
- 535 5.7.3 Authentication unsuccessful
```

> 5.7.57 Client not authenticated to send mail

Reviewer: ```suggestion
- 5.7.57 Client not authenticated to send mail
```

There are a few things you should check:

1. Enable Client SMTP submission on the licensed mailbox being used:

>    From Microsoft 365 Admin Center, go to Active Users and select the user

Reviewer: ```suggestion
   From Microsoft 365 Admin Center, go to Active Users and select the user.
```

>    Go to Mail tab

Reviewer: ```suggestion
   Go to the Mail tab.
```

>    In the “Email apps” section, click on “Manage email apps”

Reviewer: ```suggestion
   In the “Email apps” section, select **Manage email apps**.
```

>    Verify the “Authenticated SMTP” setting is checked (enabled)

Reviewer: ```suggestion
   Verify that the “Authenticated SMTP” setting is checked (enabled).
```

>    Click Save changes.

Reviewer: ```suggestion
   Select **Save changes**.
```

> 2. Disable Multi Factor Authentication (MFA) on the licensed mailbox being used:

Reviewer: ```suggestion
2. Disable Multi-Factor Authentication (MFA) on the licensed mailbox being used:
```

>    In the Microsoft 365 Admin Center, in the left navigation menu choose Users > Active users.
>    On the Active users page, choose Multi-factor authentication.
>    On the multi-factor authentication page, select the user and disable the Multi-Factor auth status.

Reviewer: ```suggestion
   - In the Microsoft 365 admin center, in the left navigation menu, choose **Users** > **Active users**.
   - On the Active users page, choose **Multi-Factor Authentication**.
   - On the Multi-Factor Authentication page, select the user and disable the Multi-Factor Authentication status.
```

> 3. Disable the [Azure Security Defaults](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/concept-fundamentals-security-defaults) by toggling the “Enable Security Defaults” to “No”:

Reviewer: ```suggestion
3. Disable the [Azure Security Defaults](/azure/active-directory/fundamentals/concept-fundamentals-security-defaults) by toggling the **Enable Security Defaults** to **No**:
```

>    Sign in to the Azure portal as a Security administrator, Conditional Access administrator, or Global administrator.
>    Browse to Azure Active Directory > Properties.
>    Select Manage security defaults.
>    Set the Enable security defaults toggle to No.
>    Select Save.

Reviewer: ```suggestion
   - Sign in to the Azure portal as a Security administrator, Conditional Access administrator, or Global administrator.
   - Browse to Azure Active Directory > Properties.
   - Select **Manage security defaults**.
   - Set the **Enable security defaults** toggle to **No**.
   - Select **Save**.
```
